yeonjae/programmers-0/직사각형넓이구하기.js

```diff
@@ -0,0 +1,15 @@
+function solution(dots) {
+  const width = [];
+  const height = [];
+  let rectangleArea = 0;
+
+  for (let i = 0; i < dots.length; i++) {
+    width.push(dots[i][0]);
+    height.push(dots[i][1]);
+
+    rectangleArea =
+      (Math.max(...width) - Math.min(...width)) *
+      (Math.max(...height) - Math.min(...height));
+  }
+  return rectangleArea;
+}
```

Comment:
```suggestion
function solution(dots) {
  const width = dots.map(el => el[0]);
  const height = dots.map(el => el[1]);

  return (
    (Math.max(...width) - Math.min(...width)) *
    (Math.max(...height) - Math.min(...height))
  );
}
```
- 연산 횟수는 조금 증가하긴 하지만 매번 푸쉬하는게 아닌 map을 통한 깔끔한 풀이도 가능하니 참고해주세요 :)

Reply:
넵 피드백 감사합니다!